crates/mfem/.rustfmt.toml

```diff
@@ -0,0 +1 @@
+max_width = 80
```

Comment:
The default setting for this is 100, and I'd prefer to keep that.
It's nice to be able to see more of the code at once.

Reply:
Ok.  In my setup, I personally prefer 80 chars because I can have 2 non-overlapping editor windows side by side which is convenient.